clients/src/test/java/org/apache/kafka/test/TestSslUtils.java

```diff
@@ -56,6 +56,24 @@
 
 public class TestSslUtils {
 
+    /*
+     * Simulates a CA signing certificates
+     */
+    static KeyPair issuerKP;
```

Comment:
We could remove this definition (and simply rely on the local one) since it doesn't seem to be used anywhere else.